llvm/test/Transforms/GVNSink/int_sideeffect.ll

```diff
@@ -28,3 +28,27 @@ if.end:
   ret float %phi
 }
 
+; CHECK-LABEL: scalarsSinkingReverse
```

Comment:
This test is just a reverse of the test above to ensure GVNSink doesn't depend on the order of branches.

Reply:
can you add that detail as a comment in the test. This seems like something that could get lost and cause the test to be removed.